src/Carbon/Lang/uz.php

```diff
@@ -48,8 +49,8 @@
     's' => ':count с',
     'ago' => ':time олдин',
     'from_now' => 'Якин :time ичида',
-    'after' => ':time пас аз он',
-    'before' => ':time пеш аз он',
+    'after' => ':time дан кейин',
+    'before' => ':time олдин',
```

Comment:
This translation is the same as "ago", is there any way to distinguish both?

before is relative: we will leave the office 2 hours before the night (both event are in the future but the first is before the second)
ago is absolute: we leaved the office 2 hours ago (= 2 hours before now).